src/index.js

```diff
@@ -1,59 +1,57 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import Payment from 'payment';
+import creditCardType, { types as cardTypes } from 'credit-card-type';
+import luhn from 'luhn';
+
+import { dankort, laser, visaElectron } from './cardTypes';
+import { sanitizeNumber } from './utils';
 
 class ReactCreditCards extends React.Component {
   constructor(props) {
     super(props);
 
-    this.setCards();
+    creditCardType.updateCard(cardTypes.MAESTRO, {
```

Comment:
this Maestro update and the Dankort addition can be removed if/when this one: https://github.com/braintree/credit-card-type/pull/106 gets merged on credit-card-type